File: COTE/LeetCode/easy/CheckIfNAndItsDouble.py
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        print(arr)
        for i in range(len(arr)-1):
            for j in range(i+1, len(arr)):
                if arr[i]==arr[j]*2 or arr[j]==arr[i]*2:
                    return True
        return False


# In case of [0,0] -> print(i, arr.index(arr[i]*2)) -> 1,0✨
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        print(arr)
        for i in range(len(arr)):
            if arr[i]*2 in arr and arr.index(arr[i]*2)!=i:
                return True
        return False

File: COTE/LeetCode/easy/CheckIfNandItsDouble.js
//https://leetcode.com/explore/learn/card/fun-with-arrays/527/searching-for-items-in-an-array/3250/

/**
 * @param {number[]} arr
 * @return {boolean}
 */
 var checkIfExist = function(arr) {
   for(let i=0; i<arr.length-1; i++){
       for(let j=i+1; j<arr.length; j++){
           if(arr[i]*2==arr[j] ||arr[j]*2==arr[i]){
               return true
           }
       }
   }
   return false
};